modules/workspace.nix

```diff
@@ -31,18 +31,29 @@ let
     type = 6;
   };
 
+  mouseActions = {
+    applicationLauncher = "org.kde.applauncher";
+    contextMenu = "org.kde.contextmenu";
+    paste = "org.kde.paste";
+    switchActivity = "switchactivity";
+    switchVirtualDesktop = "org.kde.switchdesktop";
+    switchWindow = "switchwindow";
+  };
+
+  mouseActionNames = lib.types.enum (builtins.attrNames mouseActions);
```

Comment:
I would rename to mouseActionEnum since this is an enum type, and the name indicates more of a list (to me anyway).